media/css/firefox/welcome.scss

```diff
@@ -360,3 +367,167 @@ $image-path: '/media/protocol/img';
         display: none;
     }
 }
+
+.welcome-page8.variation-2-image,
+.welcome-page8.variation-3-animation {
+    .mzp-l-content {
+        padding: 24px 52px;
+    }
+
+    .mzp-c-hero-title {
+        max-width: $content-md;
+        margin: 0 auto;
+    }
+}
+
+.image-container {
+    margin: 35px 0 70px 0;
+
+    .image-wrapper {
+        display: inline-block;
+        vertical-align: top;
+        margin: 0 50px 0 50px;
+        position: relative;
+
+        // This test is not being performed in any RTL locales
+        &:first-of-type {
+            margin: 0 50px 0 0;
+        }
+
+        &:last-child {
+            margin: 0 0 0 50px;
+        }
+
+        p {
+            @include font-size(17px);
+            max-width: 250px;
+            margin: 0 auto;
+        }
+
+        img {
+            vertical-align: top;
+            height: 290px;
+            width: auto;
+        }
+
+        video {
+            vertical-align: top;
+            pointer-events: none;
+        }
+    }
+
+    #arrow-right,
+    #arrow-down {
+        position: absolute;
+    }
+
+    #arrow-right {
+        top: 110px;
+        right: -90px;
+        height: 85px;
+    }
+
+    #arrow-down {
+        display: none;
+    }
+}
+
+@media (max-width: 1190px) {
+    .welcome-page8.variation-2-image {
+        .image-container {
+            .image-wrapper {
+                margin: 0 30px 0 30px;
+
+                &:first-of-type {
+                    margin: 0 30px 0 0;
+                }
+
+                &:last-child {
+                    margin: 0 0 0 30px;
+                }
+
+                p {
+                    @include font-size(16px);
+                    max-width: 178px;
+                }
+
+                img {
+                    height: 178px;
+                }
+
+                #arrow-right {
+                    top: 62px;
+                    right: -55px;
+                    height: 60px;
+                }
+            }
+        }
+    }
+}
+
+.welcome-page8.variation-2-image {
+    .image-container {
+        @media (max-width: 767px) {
+            max-width: 300px;
+            margin: 5px auto 70px auto;
+
+            #arrow-right {
+                display: none;
+            }
+
+            .image-wrapper {
+                margin: 0;
+
+                &:last-child {
+                    margin: 0;
+                }
+
+                &:first-of-type {
+                    margin: 0 0 50px 0;
+                }
+
+                img {
+                    height: 250px;
+                }
+
+                p {
+                    max-width: 230px;
+                }
+            }
+
+            #arrow-down {
+                display: block;
+                bottom: -63px;
+                left: 84px;
+                height: 45px;
+            }
+        }
+    }
+}
+
+.welcome-page8.variation-3-animation {
+    .image-container {
+        margin: 35px auto 70px auto;
+        max-width: 1100px;
```

Comment:
The video is 1000px wide and the extra width here is making the outer two text blocks misalign with the circles above them... only a cosmetic thing, but the off-centeredness looks odd. I think just setting this to 1000px will help keep them lined up better, though it's still not perfect, and still does some other odd things at smaller breakpoints. I think if we really want to line things up neatly with the video we might have to rework the layout of those text blocks.

Reply:
The right & left circles still look a little offline with the copy below them, but I wouldn't necessarily call it a blocker for an experiment:

![image](https://user-images.githubusercontent.com/400117/87645074-e6aca880-c744-11ea-89ff-52b40a8516b1.png)

Reply:
At small viewport sizes the three columns also become too squashed, but again given that this is a desktop centric experiment I wouldn't call it a blocker:

![image](https://user-images.githubusercontent.com/400117/87645363-4905a900-c745-11ea-8f01-d18fc08297f8.png)

Reply:
yeah, I noticed this, but I'm okay with it in this case.